src/features/auth/api/members.ts

```diff
@@ -0,0 +1,25 @@
+import { createRequest } from './instance';
+
+export const requestSignin = ({ email, password }: { email: string; password: string }) =>
+  createRequest<{ success: boolean; code: number }>({
+    method: 'POST',
+    endpoint: '/members/login',
+    body: { email: email, password: password },
+  });
+
+export const requestSignUp = ({
+  name,
+  email,
+  password,
+  position,
+}: {
+  name: string;
+  email: string;
+  password: string;
+  position: string;
+}) =>
+  createRequest<{ success: boolean; code: number; message: string }>({
+    method: 'POST',
+    endpoint: '/members/login',
+    body: { name: name, email: email, password: password, position: position },
```

Comment:
다음과 같이 변경하면 더욱 깔끔할 수 있을거 같습니다~!! 👍

```suggestion
    body: { name, email, password, position },
```